src/XrdTls/XrdTlsContext.hh

```diff
@@ -19,7 +19,7 @@
 //------------------------------------------------------------------------------
 
 #include <cstdint>
-//#include <string>
+#include <string>
```

Comment:
What make <string> be required now? Mostly out of curiosity.

Reply:
there's a compiler error when including XrdTlsContext.hh in XrdCryptoLite_bf32.cc without it. There are std::string used in XrdTlsContext.hh, so it seemed to make sense it was required, I suppose it's not previously been needed because elsewhere when XrdTlsContext.hh was included something else had previously included string. e.g. this is the error:

```
In file included from /data/dhsmith/xrootd-smithdh-repo/src/XrdCrypto/XrdCryptoLite_bf32.cc:46:
/data/dhsmith/xrootd-smithdh-repo/src/XrdTls/XrdTlsContext.hh:75:13: error: ‘string’ in namespace ‘std’ does not name a type
       {std::string cert;   //!< -> certificate path.
```
etc.

Reply:
I think the underlying issue here is that the legacy line
#if __cplusplus >= 201103L
is still there. We require the use of at least gcc 7 and there really is no need to use gcc homebrew atomics.